src/actions/incidentActions.js

```diff
@@ -151,5 +151,5 @@ export const updateIncidentCreationInput = (input) => ({
 
 export const duplicateIncident = (ticketId) => (dispatch) => {
     dispatch(createIncidentActionSet(ticketId, {}).fail({ message: 'An incident already exists for this ticket'}))
-    dispatch(ticketActions.updateIncidentQuery(ticketId))
+    dispatch(ticketActions.updateTicketQuery(ticketId))
```

Comment:
I don't see why this revision needs to occur within this propType work. Is this related? If not, please restore to the master version.

Reply:
There is no updateIncidentQuery in ticketActions.  Reverted back to master for now.  Can be cleaned up later.